### ISSUE TITLE
Can't hide menu bar

### ISSUE THREAD

**Issue by iza001** [puneetsl/lotion#112]

**Comment by corebots** [comment]
Indeed, only thing I'm missing is a way to hide the permanent menu bar on top. Would be cool if like other electron apps we could toggle it by pressing ALT.
